Source/WebCore/Modules/mediastream/gstreamer/GStreamerStatsCollector.cpp

```diff
@@ -142,7 +142,7 @@ RTCStatsReport::InboundRtpStreamStats::InboundRtpStreamStats(const GstStructure*
     frameHeight = gstStructureGet<unsigned>(structure, "frame-height"_s);
 
     if (auto identifier = gstStructureGetString(structure, "track-identifier"_s))
-        trackIdentifier = identifier.toString();
+        trackIdentifier = String(identifier.span());
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?